src/webgl/ShaderProgram.ts

```diff
@@ -164,7 +164,12 @@ export default class ShaderProgram<ValueType = any> {
     // and the formula to be used inside shader
     private static shaderBlendEq = {
         [BlendMode.MAXIMUM]: "max(color, texture2D(u_srcTexture, v_position))",
+        [BlendMode.MINIMUM]: "min(color, texture2D(u_srcTexture, v_position))",
         [BlendMode.MULTIPLY]: "clamp(color * texture2D(u_srcTexture, v_position) * 256.0, 0.0, 1.0)",
+        // [BlendMode.XOR]: "uint3(color) ^ uint3(getSrcColor())",
+        // [BlendMode.EVERY_OTHER_PIXEL]: "mix(color, getSrcColor(), float(uint(dot(v_position, u_resolution-1.0)) & 1u))",
+        // [BlendMode.EVERY_OTHER_LINE]: "mix(color, getSrcColor(), float(uint(v_position.y * u_resolution.y) & 1u))",
```

Comment:
Did you mean to un-comment these? If not, let's create tickets for these instead of leaving them commented out in the code.